contracts/src/utils/pausable.rs

```diff
@@ -9,6 +9,10 @@
 //!
 //! Note that they will not be pausable by simply including this module,
 //! only once the modifiers are put in place.
+//!
+//! Note that [`Pausable::pause`] and [`Pausable::unpause`] methods are not
+//! exposed by default.
+//! You should expose them manually in your contract's abi.
```

Comment:
```suggestion
//! You should expose them manually in your contract.
```

Reply:
Isn't previous way more clear for novice library user?
wdt @ggonzalez94 ?

Reply:
I also prefer to keep the explicit word abi.